src/scenes/UserGalleryPage/QRCode.tsx

```diff
@@ -0,0 +1,207 @@
+            radial: false,
+            color1: '#6a1a4c',
+            color2: '#6a1a4c',
+            rotation: '0',
+          },
+        },
+        cornersSquareOptions: { type: '', color: '#000000' },
+        cornersSquareOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#000000',
+            color2: '#000000',
+            rotation: '0',
+          },
+        },
+        cornersDotOptions: { type: '', color: '#000000' },
+        cornersDotOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#000000',
+            color2: '#000000',
+            rotation: '0',
+          },
+        },
+        backgroundOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#ffffff',
+            color2: '#ffffff',
+            rotation: '0',
+          },
+        },
+      });
+
+      qrCode.append(ref.current);
+      setQRCodeExists(true);
+    }
+  }, [qrCodeExists, username]);
+
+  return (
+    <>
+      <StyledFullScreenQR showQRCode={showQRCode}>
+        <StyledCloseButton
+          onClick={() => {
+            setShowQRCode(false);
+          }}
+        >
+          <CloseIcon />
+        </StyledCloseButton>
+        <StyledQRWrapper ref={ref} />
+        <Spacer height={24} />
+        <StyledUsernameText>{username}</StyledUsernameText>
+        <StyledHelperText>Scan to open {username}'s gallery in a new browser tab.</StyledHelperText>
+      </StyledFullScreenQR>
+      <StyledButton onClick={handleClick} title="Open QR code">
+        <QRIcon />
+      </StyledButton>
+    </>
+  );
+}
+
+const StyledButton = styled.button`
+  background: none;
+  border: 0;
+  cursor: pointer;
+  padding: 0;
+  height: 24px;
+  width: 24px;
+  margin-right: 8px;
+
+  & svg path {
+    stroke: ${colors.offBlack};
+  }
+`;
+
+const StyledFullScreenQR = styled.div<{ showQRCode: boolean }>`
+  pointer-events: ${({ showQRCode }) => (showQRCode ? 'all' : 'none')};
+  opacity: ${({ showQRCode }) => (showQRCode ? 1 : 0)};
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background: white;
+  z-index: 10;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  place-items: center;
+
+  transition: opacity 400ms ease;
+`;
+
+const StyledCloseButton = styled.button`
+  position: absolute;
+  top: 27px;
+  right: 27px;
+  font-size: 1rem;
+
+  background: none;
+  border: 0;
+  cursor: pointer;
+  padding: 0;
+`;
+
+const StyledUsernameText = styled.p`
+  font-family: GT Alpina;
+  font-size: 24px;
+  font-weight: 300;
+  line-height: 28px;
+  letter-spacing: -0.04em;
+  text-align: left;
+`;
+
+const StyledHelperText = styled.p`
```

Comment:
we can use the BaseM component for this!

If you click into this component in Figma, you'll see it has a Text Style (BaseM):
![CleanShot 2022-06-08 at 22 09 40](https://user-images.githubusercontent.com/80802871/172624972-b0f453d1-ac7f-44ac-a6c2-b3dc86e86821.png)


These are text styles defined in our Design System, and we have corresponding Text components defined in `Text.tsx` that you can use! You'll see a bunch in there like `TitleL`, `TitleM`, `BaseXL` etc. This should make it super easy to add text according to the spec. The goal is to rarely have to think about styling text per component.

Looks like you can use `TitleM` instead of `StyledUsernameText` too. `TitleM` is a bit special, and is italicized by default - if you wrap the text in <strong>s then it'll be non-italic